src/components/forms/Checkbox/Checkbox.js

```diff
@@ -40,10 +40,11 @@ const Checkbox = ({
         invalid={invalid}
         required={isRequired}
         appearance="custom"
+        aria-describedby={invalid ? `${id}-error-message` : undefined}
         {...inputProps}
       />
       <HelpText>{description}</HelpText>
-      {touched && <ValidationErrors error={error} />}
+      {touched && <ValidationErrors error={error} inputId={id} />}
```

Comment:
isn't it more foo-proof to compute the error message ID once and pass it down to the component entirely, instead of only passing the `inputId`?

```jsx
const errorMessageId = invalid ? `${id}-error-message` : undefined;

<Component
  aria-describedby={errorMessageId}
>
    {touched && <ValidationErrors error={error} id={errorMessageId} />}
</Component>
```

Reply:
Ahh.. yeah that would be better... Will fix